ui/src/app/cohort-search/overview/overview.component.html

```diff
@@ -72,16 +72,23 @@ <h2>Save New Cohort</h2>
         [class.show]="(name.dirty || name.touched) && cohortForm.invalid">
         <em>*A Cohort Name is Required</em>
       </label>
-      <input
-        name="name"
-        id="name-control"
-        class="control"
-        placeholder="Cohort Name"
-        formControlName="name"
-        type="text"
-        [class.invalid]="(name.dirty || name.touched) && cohortForm.invalid"
-        (input)="showConflictError = false"
-        autofocus>
+      <div class="name-area" style="position: relative;">
+        <input
+          name="name"
+          id="name-control"
+          class="control"
+          placeholder="Cohort Name"
+          formControlName="name"
+          type="text"
+          [class.invalid]="(name.dirty || name.touched) && cohortForm.invalid"
+          (input)="showConflictError = false"
+          [ngClass]="{alreadyTaken: showConflictError}"
```

Comment:
nit: Should we use class binding here to be consistent with the above class on line 83?
`[class.alreadyTaken]="showConflictError"`

Or maybe set them both using ngClass?
`[ngClass]="{invalid: (name.dirty || name.touched) && cohortForm.invalid, alreadyTaken: showConflictError}"`

Reply:
Sounds good, will convert over.